test/deprecated/ReflectionFormalNames.chpl

```diff
@@ -0,0 +1,107 @@
+use Reflection;
```

Comment:
I'd put all the ones that should work under `test/library/standard/Reflection` so we don't lose testing of them when we remove the old versions